code/build/template/build/premake4.lua

```diff
@@ -97,6 +97,10 @@ solution "[name]"
 
     flags
     {
+[+python
+        "C11",]
+[+imgui +sndh

```

Comment:
This replaces the C++ version and linking flags later in the file for these extensions.